src/%ZPM/PackageManager/Client/PublishManager.cls

```diff
@@ -0,0 +1,74 @@
+Class %ZPM.PackageManager.Client.PublishManager Extends %ZPM.PackageManager.Core.Singleton
+{
+
+Parameter NAMESPACESCOPE As BOOLEAN = 1;
+
+/// Avoids constantly reinstantiating the package service objects, which may be expensive.
+Property ServiceCache As array Of %ZPM.PackageManager.Core.IPackageService;
```

Comment:
Should be of type %ZPM.PackageManager.Core.IPublishService

Reply:
fixed